faker/providers/company/fil_PH/__init__.py

```diff
@@ -0,0 +1,86 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from collections import OrderedDict
+from ..en_PH import Provider as EnPhProvider
+
+
+class Provider(EnPhProvider):
+    """
+    Provider for company names for fil_PH locale
+
+    Companies in the Philippines rarely have Filipino names, and when they do, the English name is usually used way more
+    frequently by the locals. In some cases, the Filipino names are more like in Taglish, so for the purposes of this
+    provider, only English company names will be generated for this locale.
+
+    Company and brand taglines in pure Filipino, however, are much more common, so this provider will generate catch
+    phrases in pure Filipino randomly alongside the English ones.
+    """
+
+    catch_phrase_formats = OrderedDict([
+        ('{{english_catch_phrase}}', 0.64),
+        ('Ang {{random_noun_ish_good_trait}} ng {{random_object_of_concern}}!', 0.12),
+        ('Serbisyong {{random_good_service_adjective}} para sa {{random_object_of_concern}}!', 0.12),
+        ('Kahit kailan, {{random_good_service_adjective_chain}}!', 0.12),
+    ])
+    noun_ish_good_traits = (
+        'bida',
+        'ginhawa',
+        'haligi',
+        'karangalan',
+        'lingkod',
+        'liwanag',
+        'numero_uno',
```

Comment:
is this supposed to use an underscore rather than a hyphen?

Reply:
It should be a whitespace. Derp.